spec/rails_helper.rb

```diff
@@ -10,7 +10,7 @@
 $LOAD_PATH << Rails.root.join("spec/support/page_fragments")
 Dir[
   Rails.root.join(File.join("spec", "support", "**", "*.rb"))
-].reject { |f| f.include?("page_fragments/") }
+].reject { |f| f.include?("page_fragments/") || f.include?("factories/") }
```

Comment:
hmm do we need this? I am now even a bit lost why we "don't want to include the page fragments" here? I suppose the spec/support/page_fragments dir is on the load path from line 10 - what would happen if we didn't reject?

by the way I am now all for the `spec/support/factories` directory you used above :)

Reply:
Good question. When I didn't include this, the test suite broke. I suspect some files were being included twice (because of my line above and this one). 

Now thinking back on it, maybe my line wasn't necessary (if the original one here already included the folder). I might have followed the instructions from Factory Bot a bit too blindly.